src/modules/swap/containers/SwapModals/index.tsx

```diff
@@ -23,6 +25,7 @@ export const SwapModals = React.memo(function (props: SwapModalsProps) {
       {/*<CowSubsidyModal isOpen={showCowSubsidyModal} onDismiss={closeModals} /> */}
       {<ConfirmSwapModalSetup {...confirmSwapProps} />}
       {showNativeWrapModal && <EthFlowModal {...ethFlowProps} />}
+      {!showNativeWrapModal && <SurplusModalSetup />}
```

Comment:
Is this condition correct? `!showNativeWrapModal`

Reply:
Yes, I just want to make sure it's not displayed by accident when EthFlowModal is active